src/View/View.php

```diff
@@ -393,10 +437,34 @@ public function layoutPath($path = null)
     }
 
     /**
+     * Get the current state of auto layout.
+     *
+     * @return bool
+     */
+    public function getAutoLayout()
+    {
+        return $this->autoLayout;
+    }
+
+    /**
      * Turns on or off CakePHP's conventional mode of applying layout files.
      * On by default. Setting to off means that layouts will not be
      * automatically applied to rendered templates.
      *
+     * @param bool $autoLayout Boolean to turn on/off.
+     * @return void
+     */
+    public function setAutoLayout($autoLayout)
+    {
```

Comment:
Yep. I also renamed the getter to `isAutoLayoutEnabled()` as it is done this way in the `ViewBuilder` class.